### ISSUE TITLE
Democracy

### ISSUE THREAD

**Issue by developincognito** [ChainedDev/CityLife#768]
There are certain people within the city that have power due to government intervention but it doesn't reflect the will of the people. Thus, I believe the people should vote for who should be the high ranking official of:
* mechanics
* d8
* doj
* etc

**Comment by kimoo77** [comment]
So are you trying to say that people need to vote for CEO's in these companies ?? If that's what you're trying to say I don't think so that even should be a thing why would random people vote for a CEO of D8 when they don't even work there maybe yea the people that work there vote but not random people because the people that work there should be comfortable with who their CEO is. Whereas for mechanics every shop has it's own Bosses and there are higher people that look after all these shops and they are doing a pretty good job, for DOJ I can say yea the judge isn't always around  24/7 and they need to give powers to some other higher ranking lawyers to get the job done faster. If you meant something else then tell me.

**Comment by TimmyTwoStep** [comment]
I'm just having a hard time understanding what you're trying to say when you mentioned "government intervention" I'm guessing you mean people being staff and in positions that are the top of whitelisted jobs, from my experience, I can say that people being staff doesn't really hold anything inside of those jobs, EMS and D8 for example, I can be wrong, but I am pretty sure 3 out of the top 4 of EMS are not staff members, and D8s CEO isn't staff. Now it MAY look like staff are made higher ups in their jobs just because they are staff, but I can promise you, things are not like that. It just happens to be that most of the top positions in jobs are staff members. Staff will only interventive with whitelisted jobs and their top positions if needed. 

I don't think any vote should take place, that is just me personally speaking, I don't like politics. Most people who are higher ups in their jobs worked for it, staff or not. Mechanics shops have their own bosses, D8 currently is having issues with just being able to do their job and its task. DOJ is going to be a process unfortunately, I can see it eventually coming around, no idea how long that would take. But nonetheless, please stop referring to staff as "government" as it is against the rules to do so in character, it sounds weird, and doesn't make any sense. Some staff members would be roleplaying as some gang banger or a biker and people would want to ask if they are "government" when it doesn't make any sense in roleplay and is just awkward  to hear.

**Comment by thejorhanstahl** [comment]
Pretty sure You haven't been here long, seeing you say that people got power via staff intervention. 
Many of the people were not even staff at the time of them getting those ranks, and applied later down the line. 
Jobs were never and won't be judged or changed through some weird vote because someone wats to get some power, unless job is being abused no such thing will happen. 
All jobs were reached via RP means.

If you would like to name someone who got a job through staff means, dm myself.
tl;dr
Suggestion straight to trash, no such vote will be held. Positions are held via RP not "govt interventions".

**Comment by developincognito** [comment]
Please explain to me how the first person initially gained the power to grant gun licenses, activate inactive doors, sell vehicles, jail people, etc?

These abilities were initially granted to individuals by a member of staff either through roleplay or because their friends in real life... unless you're going to tell me some sentient AI granted it to them.

I'm not saying that it isn't possible for someone to rise/RP through the ranks and eventually have the proverbial baton passed to them.

---

You're a staff member yourself... Is it at all possible that you're bias?

Are you afraid that if a vote occurred that you wouldn't be selected for the role you currently have and thus not have the power you're trying to maintain? If not, then why would you care?

---

Lastly, just because you've been around longer doesn't necessarily suggest your ideas about how the server should go are more right than myself or someone with less time than yourself.

**Comment by developincognito** [comment]
Timmy, you made quite a few good points.

My only question is how would you refer to decisions made in the city that we don't have control over within RP?
